### PR TITLE
RavenDB-16257 Studio for Cluster-wide disable index

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts [hunk 1]
```diff
@@ -185,6 +190,9 @@ class indexes extends viewModelBase {
     }
 
     private initObservables() {
+        this.localNodeTag(this.clusterManager.localNodeTag());
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts]: this.localNodeTag = this.clusterManager.localNodeTag

then you link them and avoid creating new object ;-)

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts [hunk 2]
```diff
@@ -185,6 +190,9 @@ class indexes extends viewModelBase {
...
+        this.isCluster = ko.pureComputed(() => this.clusterManager.topology().nodes().length > 1);
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts]: clusterTopologyManager has nodesCount, please use it.

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts]: done

File: src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/indexes/indexes.ts]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**